duckduckgo/duckduckgo-privacy-extension
# Add Chrome MV3 build target

Add a separate build target for Chrome MV3 to aid us during the
transition between Chrome MV2 and Chrome MV3. Note that this MV3 is
broken so far and will need a number of follow-up changes.

**Reviewer:**
**CC:**

## Steps to test this PR:
1. This should not change anything in the existing extension builds, so I don't think anything needs to be tested. But if you can try building the MV3 extension with `npm run 

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
